internal/testutils/xds/e2e/setup_certs.go

```diff
@@ -0,0 +1,98 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
```

Comment:
:cry:  why didn't it detect these as moves?  Is it possible to fix?

Reply:
I initially did this as a `git mv`. But then realized that we had some commonly used helper functions like this which were defined in some _test.go files (probably the file where it was first defined). I thought it would be better to move these out into separate files which only have related helpers.

Reply:
Ah, ok.. I think in that case, since it's a brand new file, the (C) date should be 2022.